### DISCUSSION TITLE
Help needed: How to set up access to a Partitioned Pandas.CSVDataset using SFTP/SSH?

### DISCUSSION THREAD

**Discussion by drdebian** [kedro-org/kedro#1198] [Idea]
Hi there, 

new Kedro user here. I'm trying to set up a data pipeline where the first step is to retrieve data from a Linux host which is only accessible via SSH. On it, there is a directory full of temporally partitioned CSV files, which I would like to read in via a Kedro pipeline for further analysis.

Now, I've read through the (otherwise great) tutorials and documentation numerous times, but I'm still at a loss on how to set up access to CSV files through SFTP and what the right fsspec-parameters are to put in base/catalog.yml or local/credentials.yml. Can anyone provide a working example, where one would put hostname, username/password and remote directory information properly?

Any help would be greatly appreciated!

**Comment by datajoely** [comment]
Hi @drdebian we have this on the [docs](https://kedro.readthedocs.io/en/latest/05_data/01_data_catalog.html), but only on the 'latest' branch under example 16:

![image](https://user-images.githubusercontent.com/35801847/152128029-7510b1d0-bbf6-4d56-bb1f-ccfbbcfbc3f9.png)

**Reply by drdebian** [reply]
Thanks a lot for the quick reply, works like a charm!

**Reply by drdebian** [reply]
![image](https://user-images.githubusercontent.com/10364070/152302315-c8d83c52-6092-44a0-b3fb-7a717e886403.png)
At first I didn't think it would work, because I get the wavy red line beneath my credentials, but it looks like that's something not properly specified in the kedro catalog JSON schema.

**Reply by datajoely** [reply]
Yeah that's a bug in the schema file - it's also not a `GCSFilesystem` path in this case. Need to look into that one.
